recipes/hsp2/meta.yaml

```diff
@@ -0,0 +1,52 @@
+{% set name = "hsp2" %}
+{% set version = "0.11.0a1" %}
```

Comment:
Can you set this to a release version?

Reply:
Or, if you can't mint a stable release, can use https://github.com/conda-forge/cfep/blob/main/cfep-05.md

Reply:
Trying to follow the guidance in https://github.com/conda-forge/cfep/blob/main/cfep-05.md and staged-recipes won't let me add a recipes/hsp2/conda_build_config.yaml because of entries in .gitignore.  Kind of in a catch22 because it seems like I could easily set this up if it was a feedstock, but can't in staged-recipes itself.

Reply:
I don't see anything preventing that in https://github.com/conda-forge/staged-recipes/blob/main/.gitignore. Maybe you have to update your fork?